### DISCUSSION TITLE
Busca de equipo Hackaton

### DISCUSSION THREAD

**Discussion by DavisonH** [Launch-X-Latam/MisionFrontEnd#512] [Team Posts]
Buenas tardes compañeros.
Ando buscando un grupo que sea humilde y tenga ganas de aprender, ya que no todos contamos con mucha experiencia en temas de desarrollo pero queremos participar en este hackaton y  así poder aprender mas cosas sobre el desarrollo y adquirir experiencia.
Correo: lx12963@innovaccion.mx

**Comment by stvnmc** [comment]
te dejo mi info por si quieres armar un grupo

stvnmc
github
.
teams
***@***.*** ***@***.***>

**Comment by Saulflores01** [comment]
Hola 
Me gustaría ser parte de tu equipo
Dejo mi correo por cualquier cosa lx04135@innovacción.mx

**Comment by May-Alvarez** [comment]
Hola, ¿Todavia tienes lugar?, me gustaria unirme mi correo es lx00319@innovaccion.mx

**Comment by Yzamora25** [comment]
Hola tengo un poco de conocimiento y me agradaria aportarlo a tu equipo, te dejo mi correo por si me das la oportunidad de formar parte lx03131@innovaccion.mx

**Comment by LaMazorca** [comment]
Hola, estoy en las mismas, no tengo mucho de haber entrado al mundo de la programación pero siempre estoy dispuesto a aprender, te dejo mi correo igual lx17111@innovaccion.mx
